### DISCUSSION TITLE
Lab Assignment #4

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#27] [Assignments]
## Due 5/4 before class 

Please use this thread to post your [lab assignment's #4](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week5/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/22S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### For example

https://albertkun.github.io/22s-asiaam-191a-sample/week5/index.html

**Comment by lindseymardona** [comment]
https://lindseymardona.github.io/22s-asiaam-191a-labs/week5/index.html

**Comment by stefuntran** [comment]
https://stefuntran.github.io/22s-asiaam-191a-labs/week%205A/index.html

**Comment by kericsp** [comment]
https://kericsp.github.io/aa191-labs/week5/index.html

**Comment by willa-21** [comment]
https://willakurland.github.io/aa191-labs/Week-5/index.html

**Reply by albertkun** [reply]
nice version of your group's survey, but why is email required? :(

**Comment by kris10fu** [comment]
https://kris10fu.github.io/22s-asiaam-191a-labs/week5/index.html

**Reply by albertkun** [reply]
great job creating your own unique survey! try to see if you can find my point! 😄

**Comment by emmashanson** [comment]
https://emmashanson.github.io/22s-asianam-191a-labs/week5/index.html

**Comment by joonyloony** [comment]
https://joonyloony.github.io/22s-191a-assignments/week5/index.html

**Comment by ntran27** [comment]
https://ntran27.github.io/22s-asiaam-191a-labs/week5/index.html

**Reply by albertkun** [reply]
good job making your own version of the survey!

**Comment by XiongHer** [comment]
https://xiongher.github.io/22s-asiaam-191a-labs/week5/index.html

**Reply by albertkun** [reply]
nice one page version of the survey!

**Comment by penelopelamb** [comment]
https://penelopelamb.github.io/22s-aas-191a-labs/week5/iceCream.html

**Reply by albertkun** [reply]
hello Lillan! your survey doesn't have permissions set up for non-UCLA emails! 
![image](https://user-images.githubusercontent.com/8574425/166633380-0678af01-5bb1-45ce-8700-1ade74b969a0.png)

**Reply by penelopelamb** [reply]
ooops how about now?

**Reply by albertkun** [reply]
sadly nope: 
![image](https://user-images.githubusercontent.com/8574425/166637701-b1ffccd0-b08d-410f-bce8-f82c17fbdef6.png)

did you update the embedded iframe link?

**Reply by penelopelamb** [reply]
now?

**Reply by albertkun** [reply]
nope, did you try this?
![image](https://user-images.githubusercontent.com/8574425/166639521-7bc30279-91fa-44e7-9c03-163a11f005ae.png)

https://support.google.com/docs/answer/160166

**Reply by penelopelamb** [reply]
how about now

**Reply by albertkun** [reply]
yay it works!! i accidentally put in two of the same addresses! but see if you can find it! :)

and feel free to edit my response!

**Comment by bittertang** [comment]
https://bittertang.github.io/22s-asiaam-191a-labs/week%205/index.html

**Reply by albertkun** [reply]
hello Ivy! your survey doesnt have viewable permissions for everyone: 
![image](https://user-images.githubusercontent.com/8574425/166633503-db2629cf-cb38-4e27-bc20-120690fe920b.png)

**Reply by bittertang** [reply]
just fixed! thanks!

**Reply by albertkun** [reply]
great, it does work now! you are welcome!

**Comment by daikis12** [comment]
https://daikis12.github.io/22s-191a-assignments/week5/index.html

**Comment by kaittli** [comment]
https://kaittli.github.io/22s-asiaam-191a-labs/week5/index.html

**Comment by ananyakodali17** [comment]
https://ananyakodali17.github.io/asian-am-191a-labs/week5/

**Reply by albertkun** [reply]
Hello! It looks like you have the wrong field in your pop-up, it should be `Location` not `Hometown`!
![image](https://user-images.githubusercontent.com/8574425/167317444-8a6ab6d2-eb56-498b-afec-2f99dbbc435e.png)

**Comment by shubhgupta2018** [comment]
https://shubhgupta2018.github.io/22s-191a-assignments/week-4/index.html

**Reply by shubhgupta2018** [reply]
I hope this survey ends homophobia <333

**Reply by albertkun** [reply]
Really creative questions! See if you can find my survey response!

**Comment by codingdianat** [comment]
https://codingdianat.github.io/asiaam/aa191-assignments/week%206/index.html

**Comment by spolcharoen** [comment]
https://spolcharoen.github.io/22s-asiaam-191a-labs/week5/index.html

**Comment by antonyblin** [comment]
https://antonyblin.github.io/aas191a-assignments/week5/index.html

**Comment by ashley-lu** [comment]
https://ashley-lu.github.io/s22-asiaam-191a-assignments/week4lab/survey.html

**Comment by angelineeow** [comment]
https://angelineeow.github.io/22s-asiaam-191a-labs/week5/survey.html

**Comment by NishuKrish** [comment]
https://nishukrish.github.io/AsianAm191A-Nish/week4/index.html
(Received extension due to being sick)

**Comment by hajovel** [comment]
https://hajovel.github.io/211S-191A-Assignments/week5/index.html
